rubygem-gssapi/rubygem-gssapi.spec

```diff
@@ -0,0 +1,112 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name gssapi
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.2.0
+Release: 4%{?dist}
```

Comment:
Start release at 1

Reply:
Because this was taken from fedora I wasn't sure if we want to start the numbering over or not. Will do